test/TESTING_GUIDELINES.md

```diff
@@ -10,6 +10,25 @@ Guide to UI tests for Web SDK.
 - `brew cask install chromedriver`
 - **Note:** if you have chromedriver already installed, make sure it's the latest version: `brew cask upgrade chromedriver`
 
+If you would like to run the tests against other browsers...
+
+Firefox:
+
+- `npm install` should take care of installing Gekodriver (Webdriver for Firefox)
+- **Note:** if you have macOS Catalina installed, you will need to run `npm install -g geckodriver`, this is due to this
+  issue https://github.com/mozilla/geckodriver/issues/1629
+
+Safari:
+
+- Simply having Safari installed on your machine is enough
+
+Microsoft Edge:
+
+1. Microsoft Edge
```

Comment:
Any specific version? Intel vs Apple chip?

Reply:
Nothing specific, this isn't like x86/M1, this is the same as Chrome/Chromedriver, i.e. if you have v90 of Chrome, you need the matching v90 of Chromedriver, if that makes sense.